src/modules/folder/folder.usecase.ts

```diff
@@ -234,12 +248,8 @@ export class FolderUseCases {
   async createFolder(
     creator: User,
     name: FolderAttributes['name'],
-    parentFolderId: FolderAttributes['id'],
+    parentFolderUuid: FolderAttributes['uuid'],
   ): Promise<Folder> {
```

Comment:
This usecase is being used for nothing in this repository, so I thought it would be good to change it to a closer (you could say is a clone) version of what we have in drive-server currently, the only difference is the use of uuid.